components/Popup_form.js

```diff
@@ -0,0 +1,138 @@
+import styled from "styled-components";
+import Close from "../public/images/close.svg";
+import Image from "next/image";
+import {setGlobalState, useGlobalState} from "../state";
+import PopupAddBlockchain from "./Popup_addblockchain";
+import AddBlockchain from "../public/images/addBlockchain.svg";
+import {useState} from "react";
+import {StyledButton} from "./Popup_bookmarked";
+import {StyledPopup} from "./Popup_addblockchain";
+
+function PopupForm(props) {
+  const [openAddBlockchain] = useGlobalState("openPopupAddBlockchain");
+  const [options, setOptions] = useState();
+  const [activities, setActivities] = useState([]);
+
+  function closePopup() {
+    setGlobalState("openForm", false);
+  }
+
+  function openPopupAddBlockchain() {
+    setGlobalState("openPopupAddBlockchain", true);
+  }
+  function handlePassData(options) {
+    setOptions(options);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    const {titel, link, blockchain, date, description} = form.elements;
+    const newActivity = {
+      id: activities.length + 1,
+      titel: titel,
+      link: link.value,
+      blockchain: blockchain.value,
+      date: date.value,
+      description: description.value,
+    };
+    setActivities(activities => {
+      return [newActivity, ...activities];
+    });
+    form.reset();
+    titel.focus();
+    console.log(titel.value);
```

Comment:
Please delete `console.log`s you use only for debugging before you send your Pull Request.

Reply:
done!